service/dash.go

```diff
@@ -12,12 +13,18 @@ import (
 	ffmpeg "github.com/u2takey/ffmpeg-go"
 )
 
-func GenerateDash(fileName string) {
+func GenerateDash(fileName string, watermarkFileName string, watermarkFileDimension map[string]int, watermarkPosition map[string]int, withWatermark bool) {
```

Comment:
can you make withWatermark 3rd argument

it will be better this way `service.GenerateDash("Test.mp4", "TestWatermark.png", false, null, null)`
in case of watermark is not required

also does it works without waterMarkFilename and dimension?

Reply:
if not can you make the fields optional

Reply:
Done